chart/flux/values.yaml

```diff
@@ -33,6 +33,11 @@ helmOperator:
     url: ""
     branch: ""
     chartsPath: ""
+    # generate a SSH key named identity: ssh-keygen -q -N "" -f /tmp/identity
```

Comment:
I wouldn't put a private key in `/tmp`, anyone could find it!

Reply:
Show I change it to `$HOME/identity` ?